apps/backend/prisma/schema.prisma

```diff
@@ -76,5 +76,6 @@ model DrinkAction {
   event      Event    @relation(fields: [eventId], references: [id])
   // userId     String
   // user       User   @relation(fields: [userId], references: [authSchId])
+  hasEffect  Boolean  @default(true)
```

Comment:
there is no need to store this in the database, it can be calculated from the current time and the `createdAt` field. Whenever you're querying the drinkActions for the BAC calculations, you can apply a filter to only query those actions that happened in the past ~12 hours

Reply:
I wanted to use this method because this way we can be sure that the alcohol has already worn off and a sure solution like this seemed more professional than an estimate which could be too little or too much but of course if you say I will change it, this was just an idea.

Reply:
let's talk about it tomorrow :D